HW_1/task_5.py

```diff
@@ -0,0 +1,21 @@
+"""
+Напишите программу, которая принимает на вход координаты двух точек и находит расстояние между ними в 2D пространстве.
+Результат округлить до сотых.
+
+Ввод: четыре значения типа <int>
+Вывод: значение типа <float>
+
+Пример:
+
+4 10
+11 5
+9.22
+"""
+from math import sqrt
+
+x_A = int(input('Введите координату точки А по оси Х: '))
+y_A = int(input('Введите координату точки А по оси Y: '))
+x_B = int(input('Введите координату точки B по оси Х: '))
+y_B = int(input('Введите координату точки B по оси Y: '))
+
+print('Расстояние между точками: ', (round(sqrt((x_A - x_B) ** 2 + (y_A - y_B) ** 2), 2)))
```

Comment:
```suggestion
print('Расстояние между точками: ', round(sqrt((x_A - x_B) ** 2 + (y_A - y_B) ** 2), 2))
```
или без библиотеки `math`
```suggestion
print('Расстояние между точками: ', round(((x_A - x_B) ** 2 + (y_A - y_B) ** 2) ** (1 / 2), 2))
```